build/target-repository/docs/rector_rules_overview.md

```diff
@@ -613,6 +626,32 @@ Replaces static::* access to private constants with self::* on final classes
 
 <br>
 
+```php
+use Rector\CodeQuality\Rector\ClassConstFetch\ConvertStaticPrivateConstantToSelfRector;
+use Rector\Config\RectorConfig;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(ConvertStaticPrivateConstantToSelfRector::class, [true]);
```

Comment:
I prefer named config instead of just `true` or `false` while we don't know what thats' for. You can look up how it configured like in one of other rule, eg: `TypedPropertyFromAssignsRector`:

https://github.com/rectorphp/rector-src/blob/c21cafaf90cab17ae26900679f70461063ab791e/rules/TypeDeclaration/Rector/Property/TypedPropertyFromAssignsRector.php#L31-L45

so than the configuration method can be:

https://github.com/rectorphp/rector-src/blob/c21cafaf90cab17ae26900679f70461063ab791e/rules/TypeDeclaration/Rector/Property/TypedPropertyFromAssignsRector.php#L56-L59

and usage can be:

https://github.com/rectorphp/rector-src/blob/c21cafaf90cab17ae26900679f70461063ab791e/rules-tests/TypeDeclaration/Rector/Property/TypedPropertyFromAssignsRector/config/inline_public_configured_rule.php#L10-L12

Reply:
Looks like I was already using it this way, the human-readable key was only missing in the test config.

Reply:
Please run command:

```
composer docs
```

Reply:
Done